packages/server/src/backfill/backfill-section-group-name.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import { CourseModel } from 'course/course.entity';
+import { Command } from 'nestjs-command';
+
+@Injectable()
+export class BackfillSectionGroupName {
+  @Command({
+    command: 'backfill:section-group-name',
+    describe:
+      'set the sectionGroupName of older courses as the same as the course name',
+    autoExit: true,
+  })
+  async fix(): Promise<void> {
+    await CourseModel.createQueryBuilder()
+      .update()
+      .set({ sectionGroupName: () => 'name' })
+      .where('sectionGroupName IS NULL')
+      .execute();
+  }
```

Comment:
should we include a log message here to indicate the update occurred? i think logging how many courses were updated is good enough or smth to that effect

Reply:
Good idea, added